packages/govern-lib/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@aragon/govern-lib",
+  "version": "1.0.0",
+  "main": "dist/index.js",
+  "typings": "dist/index.d.ts",
+  "module": "dist/govern-lib.esm.js",
+  "license": "LGPL-3.0-or-later",
+  "files": ["dist"],
+  "scripts": {
+    "start": "tsdx watch",
```

Comment:
Are we sure to use here tsdx and anywhere else just tsc? I think we should have a consistent build pipeline for all JS related packages.

Reply:
We started using tsdx in our other projects (use-wallet, use-token and use-viewport are using it now) since [the features it provides by default](https://github.com/formium/tsdx#features) are really close to what we want. I think we should keep moving every new TS library to it, or do you see a reason why not?

Reply:
I couldn’t see the requirement for it but if a transition is ongoing then definitely :)